apache2/modsecurity.c

```diff
@@ -135,13 +135,29 @@ msc_engine *modsecurity_create(apr_pool_t *mp, int processing_mode) {
     return msce;
 }
 
+static void set_lock_args(struct waf_lock_args *lock_args, char *lock_name, int lock_id, char *lock_owner) {
```

Comment:
Is it a general function? If so, could we move to the waf lock utility?

Reply:
will make it modsec specific